### PR TITLE
[auto-team-labels] Added toml requirements

### PR DESCRIPTION
**Pull request by CelianR**
### What does this PR do?

Fixes issues with the `assign-teams` job failing due to `toml` module missing (https://github.com/DataDog/datadog-agent/actions/runs/8373685728/job/22927274738?pr=23956)

### CODE REVIEW

File: tasks/libs/requirements-github.txt [hunk 1]
```diff
@@ -7,3 +7,4 @@
 cryptography==39.0.1
 PyJWT==2.4.0
 PyGithub==1.59.1
+toml>=0.10.2
```
Comment (amenasria) [inline tasks/libs/requirements-github.txt:10]:
```suggestion
toml~=0.10.2
```

**Review by amenasria — CHANGES_REQUESTED** [review]
I think it'd be safer to limit this to `0.10.x`

File: tasks/libs/requirements-github.txt (hunk 1 above)
Comment (CelianR) [reply tasks/libs/requirements-github.txt]: The tasks/requirements.txt installs tasks/libs/requirements-github.txt so it should be ok

Comment (KevinFairise2) [reply tasks/libs/requirements-github.txt]: Ah yes you're right. These `requirements.txt` files are a mess 😓

**Review by KevinFairise2 — APPROVED** [review]

**Review by amenasria — APPROVED** [review]
LGTM as long as the CI passes !

**Pull request merged**